setup.py

```diff
@@ -14,7 +14,7 @@ def make_long_description():
     description="Advanced Python Logger",
     long_description=make_long_description(),
     long_description_content_type="text/markdown",
-    version="1.0.1",
+    version="1.0.2",
```

Comment:
please don't change the version here, the repo owner should do this once the PR is merged with a dedicated commit and tag

Reply:
fixed this as well